src/components/layout/MobileNavbar.jsx

```diff
@@ -67,17 +67,30 @@ class MobileNavBar extends React.Component {
     return (
       <>
         {this.state.active === 'links' ? (
-          <a
-            href="https://koala.svsticky.nl/"
-            key="Koala"
-            target="_blank"
-            rel="noopener noreferrer" // For safety
-          >
-            <div className="sub-menu-item">
-              Koala
-              <i className="item-text icon external" />
-            </div>
-          </a>
+          <>
+            <a
```

Comment:
Adding more buttons like this is getting quite verbose. Is there a reason we don't use the normal `<Button>` component @emielvanrijn1 ?

Reply:
We (me and Niels) guess it's because normal `<Button>`'s have to much of their own styling by default. Even though it's a little verbose like this, I don't think we'll have to replicate it often or in different places, so it's fine.